Assets/Scripts/Models/Buildable/Components/Workshop.cs

```diff
@@ -364,7 +364,7 @@ private void HaulingJobForInputs(ProductionChain prodChain)
                                      false,
                                      false);
                         
-                        job.Description = string.Format("Hauling '{0}' to '{1}'", desiredInv, ParentFurniture.Name);
+                        job.Description = string.Format("Hauling '{0}' to '{1}'", desiredInv, ParentFurniture.GetName());
```

Comment:
now this GetName() looks out of place while everywhere Type is used.. that method returns LocalizationCode (why is it a method, btw.)
from GetName() method I would expect it to return already localized name